[all] Compilation error: meta-asteroid/recipes-nemomobile/mce/libmce-glib

**Describe the bug**
When trying to build an asteroidOS image using containers (specifically podman), the compilation errors out on the recipe `/asteroid/src/meta-asteroid/recipes-nemomobile/mce/libmce-glib_git.bb:do_compile`

This happened both with `bass` and `qemux86`

**To reproduce**
1. Attempt to build in a podman container using `bitbake asteroid-image`
2. Specific method:
```bash
podman run --rm -it -v "$(pwd)":/asteroid:z --userns keep-id asteroidos-toolchain bash -c "source ./prepare-build.sh bass && bitbake asteroid-image"
```

**Expected behavior**
Successful build


**Device (please complete the following information):**
 - Watch Codename: bass
 - AsteroidOS Builddate: From master
 - Version From master

**Additional context**

<details>
<summary>Build log:</summary>


```
ERROR: libmce-glib-+git-r1 do_compile: oe_runmake failed
ERROR: libmce-glib-+git-r1 do_compile: ExecutionError('/asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/temp/run.do_compile.1953', 1, None, None)
ERROR: Logfile of failure stored in: /asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/temp/log.do_compile.1953
Log data follows:
| DEBUG: Executing shell function do_compile
| NOTE: make -j 16 KEEP_SYMBOLS=1
| gdbus-codegen --generate-c-code build/com.nokia.mce.request spec/com.nokia.mce.request.xml
| gdbus-codegen --generate-c-code build/com.nokia.mce.signal spec/com.nokia.mce.signal.xml
| Traceback (most recent call last):
|   File "/asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/recipe-sysroot-native/usr/bin/gdbus-codegen", line 53, in <module>
|     from codegen import codegen_main
|   File "/asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/recipe-sysroot-native/usr/share/glib-2.0/codegen/codegen_main.py", line 29, in <module>
|     from . import dbustypes
|   File "/asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/recipe-sysroot-native/usr/share/glib-2.0/codegen/dbustypes.py", line 22, in <module>
|     from . import utils
|   File "/asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/recipe-sysroot-native/usr/share/glib-2.0/codegen/utils.py", line 22, in <module>
|     import packaging.version
| ModuleNotFoundError: No module named 'packaging'
| make: *** [Makefile:155: build/com.nokia.mce.signal.c] Error 1
| make: *** Waiting for unfinished jobs....
| Traceback (most recent call last):
|   File "/asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/recipe-sysroot-native/usr/bin/gdbus-codegen", line 53, in <module>
|     from codegen import codegen_main
|   File "/asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/recipe-sysroot-native/usr/share/glib-2.0/codegen/codegen_main.py", line 29, in <module>
|     from . import dbustypes
|   File "/asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/recipe-sysroot-native/usr/share/glib-2.0/codegen/dbustypes.py", line 22, in <module>
|     from . import utils
|   File "/asteroid/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/libmce-glib/+git/recipe-sysroot-native/usr/share/glib-2.0/codegen/utils.py", line 22, in <module>
|     import packaging.version
| ModuleNotFoundError: No module named 'packaging'
| make: *** [Makefile:155: build/com.nokia.mce.request.c] Error 1
| ERROR: oe_runmake failed
| WARNING: exit code 1 from a shell command.
ERROR: Task (/asteroid/src/meta-asteroid/recipes-nemomobile/mce/libmce-glib_git.bb:do_compile) failed with exit code '1'
WARNING: systemd-1_255.13-r0 do_install: Using /home/root as root user's home directory is not fully supported by systemd
NOTE: Tasks Summary: Attempted 3989 tasks of which 3972 didn't need to be rerun and 1 failed.

Summary: 1 task failed:
  /asteroid/src/meta-asteroid/recipes-nemomobile/mce/libmce-glib_git.bb:do_compile
Summary: There was 1 WARNING message.
Summary: There were 2 ERROR messages, returning a non-zero exit code.
```

</details>

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,9 +1,9 @@
-FROM ubuntu:focal
+FROM ubuntu:latest
 
 # Install packages required to build AsteroidOS
 # And add the en_US.utf8 locale because it is required and not installed by default in minimal Ubuntu images
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt update && apt upgrade -y && apt install -y git build-essential cpio diffstat gawk chrpath texinfo python3 python3-distutils wget shared-mime-info zstd liblz4-tool locales \
+RUN apt update && apt upgrade -y && apt install -y git build-essential cpio diffstat gawk file chrpath texinfo python3 python3-packaging wget shared-mime-info zstd liblz4-tool locales \
     && rm -rf /var/lib/apt/lists/* && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 
 ENV LANG en_US.utf8

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:latest
+FROM ubuntu:noble
 
 # Install packages required to build AsteroidOS
 # And add the en_US.utf8 locale because it is required and not installed by default in minimal Ubuntu images

